ui-tests/src/test/java/org/jenkins/plugins/pitmutation/PitMutationTest.java

```diff
@@ -0,0 +1,399 @@
+import org.jenkins.plugins.pitmutation.Views.MutationTableView;
+import org.jenkins.plugins.pitmutation.Views.PitMutation.MutationStatistics;
+import org.jenkins.plugins.pitmutation.tables.ClassComponents.ClassComponentTable;
+import org.jenkins.plugins.pitmutation.tables.ClassComponents.ClassComponentTableEntry;
+import org.jenkins.plugins.pitmutation.tables.Components.ComponentTable;
+import org.jenkins.plugins.pitmutation.tables.Components.ComponentTableEntry;
+import org.jenkins.plugins.pitmutation.tables.MutationInformation.MutationInformationTable;
+import org.jenkins.plugins.pitmutation.tables.MutationInformation.MutationInformationTableEntry;
+import org.jenkins.plugins.pitmutation.tables.MutationSource.MutationSourceTable;
+import org.jenkins.plugins.pitmutation.tables.MutationSource.MutationSourceTableEntry;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.po.Build;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@WithPlugins({"pitmutation", "pipeline-stage-step", "workflow-durable-task-step", "workflow-basic-steps",
+    "workflow-job", "workflow-scm-step", "workflow-cps"})
+public class PitMutationTest extends UiTest {
+    private final static int FIRST_ROW_INDEX = 0;
+    private final static int LAST_ROW_INDEX = 15;
+    private final static String ROW_KEY = "42";
+    private final static String EMPTY_ID = "";
+    private final static String FIRST_LEVEL = "Aggregated Reports";
+    private final static String SECOND_LEVEL = "null";
+    private final static String THIRD_LEVEL = "edu.hm.hafner.util.PathUtil";
+
+    /**
+     * Checks the output console if the build was successful and the PitMutation plugin was used.
+     */
+    @Test
+    public void verifyConsoleOutput() {
+        Build build = createAndBuildWorkflowJob();
+        ConsoleView consoleView = new ConsoleView(build, "console");
+
+        assertThat(consoleView.getConsoleOutput()).contains("[Pipeline] pitmutation");
+        assertThat(consoleView.getConsoleOutput()).contains("Finished: SUCCESS");
+    }
+
+    /**
+     * Checks if the testExamined are displayed correctly.
+     */
+    @Test
+    public void verifyTestExamined() {
+        MutationDetailView classView = getClassView();
+
+        List<String> tests = classView.getTestsExamined();
+
+        assertThat(tests.size()).isEqualTo(21);
+        assertThat(tests.get(0)).contains("edu.hm.hafner.util.PathUtilTest", "shouldReturnFallbackOnError",
+            "[test-template-invocation:#3] (8 ms)");
+    }
+
+    /**
+     * Checks if the mutators are displayed correctly.
+     */
+    @Test
+    public void verifyMutators() {
+        MutationDetailView classView = getClassView();
+
+        List<String> mutators = classView.getActiveMutators();
+
+        assertThat(mutators.size()).isEqualTo(11);
+        assertThat(mutators.get(0)).isEqualTo("CONDITIONALS_BOUNDARY");
+    }
+
```

Comment:
In UI tests it makes more sense to reduce the number of tests (the initial setup is too slow).  So I would recommend to combine those tests with the same setup into a single test.